lib/cli/drive-selection-widget.js

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
I'm not going to comment on each one, but the JSDoc should contain their title preceded by `@summary`, the types should be capitalized (e.g: `Number`, `Object`), every annotation should contain the access type, and the type of annotation (e.g: `@function`), and each function should contain an `@example`.